[msvc] number of sections exceeded object file format limit: compile with /bigobj

Only happening in Debug mode

```

"C:\Sources\bitcoin\build_msvc\bitcoin.sln" (default target) (1) ->
"C:\Sources\bitcoin\build_msvc\libbitcoinconsensus\libbitcoinconsensus.vcxproj" (default target) (2) ->
(Lib target) ->
  sha256_sse4.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation
 that consumes this library [C:\Sources\bitcoin\build_msvc\libbitcoinconsensus\libbitcoinconsensus.vcxproj]


"C:\Sources\bitcoin\build_msvc\bitcoin.sln" (default target) (1) ->
"C:\Sources\bitcoin\build_msvc\testconsensus\testconsensus.vcxproj" (default target) (3) ->
"C:\Sources\bitcoin\build_msvc\libbitcoin_util\libbitcoin_util.vcxproj" (default target) (4) ->
  sync.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation that c
onsumes this library [C:\Sources\bitcoin\build_msvc\libbitcoin_util\libbitcoin_util.vcxproj]
  compat_strnlen.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operat
ion that consumes this library [C:\Sources\bitcoin\build_msvc\libbitcoin_util\libbitcoin_util.vcxproj]


"C:\Sources\bitcoin\build_msvc\bitcoin.sln" (default target) (1) ->
"C:\Sources\bitcoin\build_msvc\bitcoind\bitcoind.vcxproj" (default target) (6) ->
"C:\Sources\bitcoin\build_msvc\libbitcoin_wallet\libbitcoin_wallet.vcxproj" (default target) (10) ->
(ClCompile target) ->
  C:\Sources\bitcoin\src\wallet\wallet.cpp : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj [C:\Sources\bitc
oin\build_msvc\libbitcoin_wallet\libbitcoin_wallet.vcxproj]

    3 Warning(s)
    1 Error(s)
```

Probably easy to fix, will check later.

--- a/build_msvc/common.init.vcxproj
+++ b/build_msvc/common.init.vcxproj
@@ -73,6 +73,7 @@
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <AdditionalOptions>/bigobj %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
@@ -94,6 +95,7 @@
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <AdditionalOptions>/bigobj %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
   </ItemDefinitionGroup>
 

